src/season/schema/season.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+@Schema()
+export class Season {
+  @Prop({unique: true, required: true})
+  id: number;
+
+  @Prop({required: true})
+  name: string;
+
+  @Prop({required: true})
+  leagueId: number;
+
+  @Prop({required: true})
+  teamsIds: number[];
```

Comment:
Here, the property type is not a number, but an ObjectId.
[ObjectId reference](https://www.mongodb.com/docs/manual/reference/method/ObjectId/)

Reply:
It's the team id of third party that we filter teams based on it before saving to our database.